content/en/storage-providers/curio/overview.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Overview"
+description: "This section covers a overview of what Curio is, and how it relates to the Lotus-Miner"
+lead: "This section covers a overview of what Curio is, and how it relates to the Lotus-Miner"
+draft: false
+menu:
+    storage-providers:
+        parent: "curio"
+weight: 105
+toc: true
```

Comment:
This should have aliases to the old lotus-provider page so it keeps current links intact.

Reply:
Alias are required to avoid broken links in large docs. There would be no need here as these docs were self contained. I see no link to any of these pages.